Library/Formula/rdiff-backup.rb

```diff
@@ -18,6 +19,11 @@ class RdiffBackup < Formula
 
   depends_on "librsync"
 
+  patch do
+    url "https://gist.githubusercontent.com/nijikon/b9bc54002cceb395a825/raw/44574d1df25789d145e2d830cd3e6d68f2567055/rdiff-backup_librsyncmodule.c.patch"
+    sha256 "df0a223e83697fa4b0d715445ae8a3a5398d8efde6bcedf2690fbbf4eb0560e8"
+  end unless build.devel?
```

Comment:
Oh, of course. Put it in a `stable do` block then :wink:. I'm killing off `end if` and `end unless` patches.